backfill_mismatched_deposits.sql

```diff
@@ -0,0 +1,64 @@
+-- soft-delete events for overcounted deposits
+UPDATE events SET deleted_at = current_timestamp
+WHERE
+  deposit_id IN (
+  SELECT 
+    deposits.id
+  FROM
+    deposits
+  LEFT JOIN 
+    blocks
+  ON
+    deposits.block_hash = blocks.hash
+  WHERE
+    deposits.main = true AND (blocks.main IS NULL OR blocks.main = false)
```

Comment:
Do we know a case where blocks.main would be null?

Reply:
There are some cases (~1.5k) where there's no record in `blocks` that matches `deposits.block_hash`, so this is meant to capture those.